backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java

```diff
@@ -84,16 +66,16 @@ public DockerCassandraRule() {
                         .run("sed -i -e \"s/commitlog_sync_period_in_ms: 10000/commitlog_sync_period_in_ms: 9999999/\" " + CASSANDRA_YAML)
                         //auto_bootstrap should be useless when no existing data
                         .run("echo auto_bootstrap: false >> " + CASSANDRA_YAML)
-                        .run("echo \"-Xms1500M\" >> " + JVM_OPTIONS)
-                        .run("echo \"-Xmx1500M\" >> " + JVM_OPTIONS)
+                        .run("echo \"-Xms500M\" >> " + JVM_OPTIONS)
+                        .run("echo \"-Xmx500M\" >> " + JVM_OPTIONS)
```

Comment:
Downgrading memory don't seem related with the title of the commit. Could you revert this?

Reply:
oh yes, it was to make it run on my computer